uint8 UNDEFINED         =   0
uint8 VERTICAL          =   1
uint8 HORIZONTAL        =   2
uint8 LEFT              =  10
uint8 RIGHT             =  11
uint8 DOWN              =  12

Header header
geometry_msgs/Pose handle_pose
geometry_msgs/Pose hinge_pose
uint8 handle_orientation
uint8 handle_side